### PR TITLE
New Checkout error codes Dev. Guide

### PR DESCRIPTION
**Pull request by ricardoaerobr**
#### Types of changes
- [X] New content (guides, endpoints, app documentation)
- [ ] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

PR created to publish the new Checkout error codes Dev. Guide.

### CODE REVIEW

File: docs/guides/Checkout/checkout-error-codes.md [hunk 1]
```diff
@@ -0,0 +1,344 @@
+title: "Checkout error codes"
+slug: "checkout-error-codes"
+hidden: true
+createdAt: ""
+updatedAt: ""
+-------------
+
+During Checkout API calls, some error codes may appear as a message in the response body. See the meaning of each one of them:
```
Comment (karenkrieger) [inline docs/guides/Checkout/checkout-error-codes.md:8]:
```suggestion
During Checkout API requests, some error codes may appear as a message in the response body. See the meaning of each one of them:
```

File: docs/guides/Checkout/checkout-error-codes.md [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+| **CHK0290**      | `ORICommunicationError`                                                      | There was a communication error with the Routing System.                                                                                                                             |
+| **CHK0291**      | `ExternalMarketplaceCommunicationError`                                      | There was a communication error with the External Marketplace.                                                                                                                       |
+| **CHK0292**      | `LockTimeout`                                                                | The time limit for obtaining the access lock to a required resource was exceeded.                                                                                                    |
+| -----------       | ------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------                        |
```
Comment (karenkrieger) [inline docs/guides/Checkout/checkout-error-codes.md:286]: Isso aqui tá certo?

File: docs/guides/Checkout/checkout-error-codes.md (hunk 1 above)
Comment (ricardoaerobr) [reply docs/guides/Checkout/checkout-error-codes.md]: Fixed.

**Review by karenkrieger — APPROVED** [review]
LGTM

**Pull request merged**